shells.nix

```diff
@@ -97,7 +98,7 @@ in
                       fi
                       echo ⛑ Changing dir to \"$componentDir\"
                       cd "$componentDir"
-                      ${if drv ? targetSetup then "${drv.targetSetup}/bin/target-setup" else ""}
+                      ${if drv ? targetSetup then "checkTargetSetupMarkers" else ""}
```

Comment:
This should not be needed here. The setup hook should add itself as a shell hook.

Reply:
hookception, good idea.